src/pages/creer-un-jeu/mimique/2.tsx

```diff
@@ -13,6 +13,15 @@ const MimiqueStep2 = () => {
   const router = useRouter();
   const { activity, updateActivity } = React.useContext(ActivityContext);
 
+  const data = (activity?.data as MimicsData) || null;
+
+  const isValid = React.useMemo(() => {
+    if (activity !== null && activity.content.filter((c) => c.value.length > 0 && c.value !== '<p></p>\n').length === 0) {
+      return false;
+    }
+    return true;
+  }, [activity]);
```

Comment:
I'm not sure the function here does what it should. You want to check if the step 1 of the mimic activity is valid. So:
- I would rename `isValid` to `isStep1Valid`.
- I would create a function in `/src/activity-types/game.constants.ts`: `function isMimicValid(data: MimicData) { [...] }` with the logic you have here: https://github.com/parlemonde/1village/blob/4c8fd297e57c842243443e957859997169e7bf4c/src/components/selectors/MimicSelector.tsx#L34 and use it: `return isMimicValid(data.game1)`

Reply:
I did the change for this file and created a function in game.constants.ts as suggested.